components/language_filter/requirements.txt

```
@@ -0,0 +1,4 @@
+fondant
```

```suggestion
git+https://github.com/ml6team/fondant@main
```
Our build / release pipelines expect fondant to be installed from git.